lib/ad_service/query/for_display.ex

```diff
@@ -2,7 +2,28 @@ defmodule AdService.Query.ForDisplay do
   import Ecto.Query
   alias AdService.Advertisement
   alias CodeFund.Campaigns
-  alias CodeFund.Schema.{Audience, Campaign, Creative, Impression}
+  alias CodeFund.Schema.{Audience, Campaign, Creative, Impression, Property}
+
+  def fallback_ad_by_property_id(property_id) do
```

Comment:
Does this mean that the fallback campaign doesn't have to have a budget set or even be active?

Reply:
Yep.